okhttp/src/main/java/com/squareup/okhttp/MediaType.java

```diff
@@ -31,6 +31,64 @@
   private static final Pattern PARAMETER = Pattern.compile(
       ";\\s*(?:" + TOKEN + "=(?:" + TOKEN + "|" + QUOTED + "))?");
 
+  /**
+   * A {@link MediaType} constant representing the wildcard media type.
+   */
+  public static final MediaType WILDCARD = new MediaType("*/*", "*", "*", null);
+  /**
+   * A {@link MediaType} constant representing the "application/xml" media type.
+   */
+  public static final MediaType APPLICATION_XML =
+                  new MediaType("application/xml", "application", "xml", null);
+  /**
+   * A {@link MediaType} constant representing the "application/atom+xml" media type.
+   */
+  public static final MediaType APPLICATION_ATOM_XML =
```

Comment:
Fair enough, I can change that.

The media types are currently exactly the same that are also provided by the JAX-RS class [`MediaType`](http://docs.oracle.com/javaee/7/api/javax/ws/rs/core/MediaType.html).